.github/CODEOWNERS

```diff
@@ -93,6 +93,8 @@ extension/basicauthextension/                                           @open-te
 extension/bearertokenauthextension/                                     @open-telemetry/collector-contrib-approvers @jpkrohling @pavankrish123 @frzifus
 extension/encoding/jaegerencodingextension/                             @open-telemetry/collector-contrib-approvers @MovieStoreGuy @atoulme
 extension/encoding/textencodingextension/                               @open-telemetry/collector-contrib-approvers @MovieStoreGuy @atoulme
+extension/encoding/otlpencodingextension/                               @open-telemetry/collector-contrib-approvers @dao-jun
+extension/encoding/zipkinencodingextension/                             @open-telemetry/collector-contrib-approvers @dao-jun
```

Comment:
Could you add me to the code owners here as well?

@astencel-sumo as well? (If they agree)